test/tests/parallel/set1/spam_virus/amavis.bats

```diff
@@ -44,3 +75,19 @@ function teardown_file() { _default_teardown ; }
   assert_success
   assert_output --partial "= '***SPAM*** ';"
 }
+
+@test '(Amavis disabled) configuration should be correct' {
+  export CONTAINER_NAME=${CONTAINER2_NAME}
+
+  _run_in_container grep 'smtp-amavis:[127.0.0.1]:10024' /etc/postfix/main.cf
+  assert_failure
```

Comment:
`postfconf content_filter` would output `content_filter =` (along with the two warning lines), and with `-h` option just a blank line.

If you did want to apply the same change from grep here, you couldn't assert failure and would have to refute. `grep` may be the better choice :man_shrugging:

Reply:
Done in 63abdb8824ce2a73f16b66e100f71f0479967065